cds-snc/cppd-medical-report-sails
# Add overall health section

## Depends on
#20 #18  - merge them first

## Adds
- Brings back the Overall Health step
- Saves then redirects back to Conditions (for now)

## To test
- Save personal information
- Add one or more conditions
- Skip Medications
- Skip Treatments
- Submit Health form without info, note validation
- Enter some health information
- Submit form (will redirect to Conditions)
- Navigate past Medications/Treatments again to see the form populated with your previous info
- Change the info
- Submit form (will redirect back to Conditions)
- Navigate past Medications/Treatments again to see your changed information
